Update bigquery extractor docs (#278)
keboola/connection-docs: components/extractors/database/index.md

## NoSQL databases

> Unlike SQL databases, the extractors for a **NoSQL databases**, require a different configuration (except [BigQuery Extractor](/components/extractors/database/bigquery/) for [BigQuery](https://cloud.google.com/bigquery/) database which is quite similar to SQL databases, also support [advanced mode](/components/extractors/database/sqldb/)), see:

Reviewer: ```suggestion
Unlike the extractor for SQL databases, extractors for **NoSQL databases** require a different configuration (except the [BigQuery Extractor](/components/extractors/database/bigquery/) for the [BigQuery](https://cloud.google.com/bigquery/) database, which is quite similar to SQL databases and also supports the [advanced mode](/components/extractors/database/sqldb/)):
```

> - [MongoDB Extractor](/components/extractors/database/mongodb/) for [MongoDB](https://www.mongodb.com/) database and [CosmosDB MongoDB API](https://docs.microsoft.com/en-us/azure/cosmos-db/mongodb-introduction).

Reviewer: ```suggestion
- [MongoDB extractor](/components/extractors/database/mongodb/) for the [MongoDB](https://www.mongodb.com/) database and the [CosmosDB MongoDB API](https://docs.microsoft.com/en-us/azure/cosmos-db/mongodb-introduction).
```

> Unlike the extractor for SQL databases, extractors for **NoSQL databases** require a different configuration (except the [BigQuery Extractor](/components/extractors/database/bigquery/) for the [BigQuery](https://cloud.google.com/bigquery/) database, which is quite similar to SQL databases and also supports the [advanced mode](/components/extractors/database/sqldb/)):

Reviewer: ```suggestion
Unlike the extractors for SQL databases, extractors for **NoSQL databases** require a different configuration (except the [BigQuery Extractor](/components/extractors/database/bigquery/) for the [BigQuery](https://cloud.google.com/bigquery/) database, which is quite similar to SQL databases and also supports the [advanced mode](/components/extractors/database/sqldb/)):
```

> - [CosmosDB Extractor](/components/extractors/database/cosmosdb/) for [CosmosDB SQL API](https://docs.microsoft.com/en-us/azure/cosmos-db/tutorial-query-sql-api).

Reviewer: ```suggestion
- [CosmosDB extractor](/components/extractors/database/cosmosdb/) for the [CosmosDB SQL API](https://docs.microsoft.com/en-us/azure/cosmos-db/tutorial-query-sql-api).
```

> - [Azure Storage Table Extractor](/components/extractors/database/cosmosdb/) for [Azure Table storage](https://azure.microsoft.com/en-us/services/storage/tables) and [Cosmos DB Table API](https://docs.microsoft.com/en-us/azure/cosmos-db/table-introduction).

Reviewer: ```suggestion
- [Azure Storage Table extractor](/components/extractors/database/cosmosdb/) for the [Azure Table storage](https://azure.microsoft.com/en-us/services/storage/tables) and the [Cosmos DB Table API](https://docs.microsoft.com/en-us/azure/cosmos-db/table-introduction).
```
